### ISSUE TITLE
CLI Source Map issue

### ISSUE THREAD

**Issue by roikles** [sass/node-sass#573]
Trying to get sourcemaps working using the following command line args:

`node-sass --output-style compressed --source-map assets/sass/style.scss style.css`

My css file is created as is style.map.css which is also referenced in the CSS file, so far so good. When I try to use the sourcemap in Chrome (latest release) it doesn't recognise it, so I started exsperimenting.

I noticed by adding a path following the --source-map creates a sourcemap with a differnt filename e.g.

`node-sass --output-style compressed --source-map test.css.map  assets/sass/style.scss style.css`.

The new file `test.css.map` is created and has different output to the default style.css.map file.

style.css.map (created without --source-map additional filename arg) output:

```
{"version":3,"sources":["style.css"],"names":[],"mappings":"AAAA,gBAAgB,cAAc,CAAA,oBAAqB,CAAA,oBAAqB,CAAA,sHAAA,CAA8G,8GAAA,AAAC,CAAA,SAAU,cAAc,CAAA,oBAAqB,CAAA,sHAAA,CAA8G,8GAAA,AAAC,CAAA,CAAE,SAAS,CAAA,SAAU,CAAA,iBAAkB,CAAA,qBAAqB,AAAC,CAAA,iCAAkC,sBAAqB,AAAC,CAAA,QAAS,cAAa,AAAC,CAAA,0BAA2B,SAAS,UAAU,CAAA,iBAAiB,CAAC,AAAC,CAAA,0BAA2B,gBAAgB,UAAU,CAAA,iBAAiB,CAAC,AAAC,CAAA,KAAM,qBAAoB,OAAO,AAAC,CAAA,wBAAyB,WAAW,CAAA,aAAa,AAAC,CAAA,WAAY,WAAU,AAAC,CAAA,OAAQ,WAAW,CAAA,iBAAkB,CAAA,mBAAmB,AAAC,CAAA,gBAAiB,cAAa,AAAC,CAAA,kBAAmB,eAAc,AAAC,CAAA,WAAY,eAAc,AAAC,CAAA,sBAAuB,gBAAe,AAAC,CAAA,sBAAuB,UAAS,AAAC,CAAA,iCAAkC,gBAAe,AAAC,CAAA,WAAY,gBAAe,AAAC,CAAA,4CAA6C,UAAS,AAAC,CAAA,WAAY,gBAAe,AAAC,CAAA,iCAAkC,gBAAe,AAAC,CAAA,sBAAuB,UAAS,AAAC,CAAA,uBAAwB,gBAAe,AAAC,CAAA,YAAa,gBAAe,AAAC,CAAA,mEAAoE,WAAU,AAAC,CAAA,UAAW,UAAS,AAAC,CAAA,UAAW,UAAS,AAAC,CAAA,UAAW,UAAS,AAAC,CAAA,UAAW,UAAS,AAAC,CAAA,UAAW,WAAU,AAAC,CAAA,UAAW,gBAAe,AAAC,CAAA,UAAW,gBAAe,AAAC,CAAA,UAAW,gBAAe,AAAC,CAAA,UAAW,gBAAe,AAAC,CAAA,UAAW,gBAAe,AAAC,CAAA,UAAW,gBAAe,AAAC,CAAA,UAAW,WAAU,AAAC,CAAA,mBAAoB,cAAc,CAAA,UAAU,AAAC,CAAA,WAAY,qBAAoB,AAAC,CAAA,sBAAuB,sBAAqB,AAAC,CAAA,sBAAuB,gBAAe,AAAC,CAAA,iCAAkC,sBAAqB,AAAC,CAAA,WAAY,sBAAqB,AAAC,CAAA,4CAA6C,gBAAe,AAAC,CAAA,WAAY,sBAAqB,AAAC,CAAA,iCAAkC,sBAAqB,AAAC,CAAA,sBAAuB,gBAAe,AAAC,CAAA,uBAAwB,sBAAqB,AAAC,CAAA,YAAa,sBAAqB,AAAC,CAAA,YAAa,YAAW,AAAC,CAAA,UAAW,eAAe,CAAA,aAAc,CAAA,WAAY,CAAA,UAAW,CAAA,gBAAiB,CAAA,cAAe,CAAA,gBAAiB,CAAA,WAAY,CAAA,kBAAmB,CAAA,eAAe,AAAC,CAAA,yBAA0B,MAAM,qBAAoB,OAAO,AAAC,CAAA,wBAAyB,WAAW,CAAA,aAAa,AAAC,CAAA,WAAY,WAAU,AAAC,CAAA,OAAQ,WAAW,CAAA,iBAAkB,CAAA,mBAAmB,AAAC,CAAA,mBAAoB,cAAa,AAAC,CAAA,qBAAsB,eAAc,AAAC,CAAA,cAAe,eAAc,AAAC,CAAA,4BAA6B,gBAAe,AAAC,CAAA,4BAA6B,UAAS,AAAC,CAAA,0CAA2C,gBAAe,AAAC,CAAA,cAAe,gBAAe,AAAC,CAAA,wDAAyD,UAAS,AAAC,CAAA,cAAe,gBAAe,AAAC,CAAA,0CAA2C,gBAAe,AAAC,CAAA,4BAA6B,UAAS,AAAC,CAAA,6BAA8B,gBAAe,AAAC,CAAA,eAAgB,gBAAe,AAAC,CAAA,qFAAsF,WAAU,AAAC,CAAA,aAAc,UAAS,AAAC,CAAA,aAAc,UAAS,AAAC,CAAA,aAAc,UAAS,AAAC,CAAA,aAAc,UAAS,AAAC,CAAA,aAAc,WAAU,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,WAAU,AAAC,CAAA,sBAAuB,cAAc,CAAA,UAAU,AAAC,CAAA,cAAe,qBAAoB,AAAC,CAAA,4BAA6B,sBAAqB,AAAC,CAAA,4BAA6B,gBAAe,AAAC,CAAA,0CAA2C,sBAAqB,AAAC,CAAA,cAAe,sBAAqB,AAAC,CAAA,wDAAyD,gBAAe,AAAC,CAAA,cAAe,sBAAqB,AAAC,CAAA,0CAA2C,sBAAqB,AAAC,CAAA,4BAA6B,gBAAe,AAAC,CAAA,6BAA8B,sBAAqB,AAAC,CAAA,eAAgB,sBAAqB,AAAC,CAAA,eAAgB,YAAW,AAAC,CAAA,UAAW,eAAe,CAAA,aAAc,CAAA,WAAY,CAAA,UAAW,CAAA,gBAAiB,CAAA,cAAe,CAAA,gBAAiB,CAAA,WAAY,CAAA,kBAAmB,CAAA,gBAAgB,CAAC,AAAC,CAAA,0BAA2B,MAAM,qBAAoB,OAAO,AAAC,CAAA,wBAAyB,WAAW,CAAA,aAAa,AAAC,CAAA,WAAY,WAAU,AAAC,CAAA,OAAQ,WAAW,CAAA,iBAAkB,CAAA,mBAAmB,AAAC,CAAA,mBAAoB,cAAa,AAAC,CAAA,qBAAsB,eAAc,AAAC,CAAA,cAAe,eAAc,AAAC,CAAA,4BAA6B,gBAAe,AAAC,CAAA,4BAA6B,UAAS,AAAC,CAAA,0CAA2C,gBAAe,AAAC,CAAA,cAAe,gBAAe,AAAC,CAAA,wDAAyD,UAAS,AAAC,CAAA,cAAe,gBAAe,AAAC,CAAA,0CAA2C,gBAAe,AAAC,CAAA,4BAA6B,UAAS,AAAC,CAAA,6BAA8B,gBAAe,AAAC,CAAA,eAAgB,gBAAe,AAAC,CAAA,qFAAsF,WAAU,AAAC,CAAA,aAAc,UAAS,AAAC,CAAA,aAAc,UAAS,AAAC,CAAA,aAAc,UAAS,AAAC,CAAA,aAAc,UAAS,AAAC,CAAA,aAAc,WAAU,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,gBAAe,AAAC,CAAA,aAAc,WAAU,AAAC,CAAA,sBAAuB,cAAc,CAAA,UAAU,AAAC,CAAA,cAAe,qBAAoB,AAAC,CAAA,4BAA6B,sBAAqB,AAAC,CAAA,4BAA6B,gBAAe,AAAC,CAAA,0CAA2C,sBAAqB,AAAC,CAAA,cAAe,sBAAqB,AAAC,CAAA,wDAAyD,gBAAe,AAAC,CAAA,cAAe,sBAAqB,AAAC,CAAA,0CAA2C,sBAAqB,AAAC,CAAA,4BAA6B,gBAAe,AAAC,CAAA,6BAA8B,sBAAqB,AAAC,CAAA,eAAgB,sBAAqB,AAAC,CAAA,eAAgB,YAAW,AAAC,CAAA,UAAW,eAAe,CAAA,aAAc,CAAA,WAAY,CAAA,UAAW,CAAA,gBAAiB,CAAA,cAAe,CAAA,gBAAiB,CAAA,WAAY,CAAA,kBAAmB,CAAA,eAAe,CAAC,AAAC,CAAA,sBAAuB,gBAAgB,CAAA,gBAAiB,CAAA,oBAAoB,AAAC,CAAA,6EAA8E,gBAAe,AAAC,CAAA,iBAAkB,uCAAuC,CAAA,eAAgB,CAAA,oBAAoB,AAAC,CAAA,EAAG,iBAAgB,AAAC,CAAA,EAAG,iBAAgB,AAAC,CAAA,EAAG,iBAAgB,AAAC,CAAA,MAAO,iBAAgB,AAAC,CAAA,EAAG,kBAAiB,AAAC,CAAA,KAAM,mBAAkB,AAAC,CAAA,GAAI,iBAAgB,AAAC,CAAA,SAAU,kBAAiB,AAAC,CAAA,WAAY,kBAAiB,AAAC,CAAA,UAAW,iBAAgB,AAAC,CAAA,QAAS,eAAc,AAAC,CAAA,YAAa,qBAAqB,CAAA,UAAW,CAAA,QAAS,CAAA,cAAe,CAAA,WAAY,CAAA,YAAa,CAAA,iCAAiC,AAAC,CAAA,wBAAyB,qBAAoB,AAAC,CAAA,wBAAyB,gBAAe,AAAC,CAAA,OAAO,OAAO,AAAC,CAAA,0BAA2B,WAAW,CAAA,aAAa,AAAC,CAAA,YAAa,WAAU,AAAC,CAAA,KAAM,cAAa,AAAC,CAAA,qBAAsB,oBAAoB,CAAA,YAAY,AAAC,CAAA,KAAM,qBAAqB,CAAA,UAAW,CAAA,WAAY,CAAA,YAAa,CAAA,qBAAqB,AAAC,CAAA,WAAY,mBAAkB,AAAC,CAAA,QAAS,WAAW,CAAA,qBAAqB,AAAC,CAAA,cAAe,mBAAkB,AAAC,CAAA,GAAI,WAAW,CAAA,cAAe,CAAA,WAAW,AAAC,CAAA,KAAM,SAAQ,AAAC,CAAA,UAAW,WAAW,CAAA,aAAa,AAAC,CAAA,WAAY,WAAW,CAAA,aAAa,AAAC,CAAA,WAAY,eAAc,AAAC,CAAA,gBAAiB,WAAU,AAAC,CAAA,OAAQ,iBAAgB,AAAC,CAAA,gBAAiB,eAAc,AAAC,CAAA,aAAc,iBAAgB,AAAC,CAAA,YAAa,cAAa,AAAC,CAAA,WAAW,OAAQ,CAAA,gBAAgB,AAAC,CAAA,kCAAmC,WAAW,CAAA,aAAa,AAAC,CAAA,gBAAiB,WAAU,AAAC,CAAA,gBAAiB,WAAW,CAAA,UAAU,AAAC,CAAA,gBAAiB,YAAY,CAAA,QAAS,CAAA,eAAe,AAAC,CAAA,gBAAiB,WAAW,CAAA,cAAe,CAAA,gBAAgB,AAAC,CAAA,kBAAmB,qBAAoB,AAAC,CAAA,OAAQ,qBAAqB,CAAA,gBAAgB,AAAC,CAAA,2BAA4B,cAAc,CAAA,cAAe,CAAA,oBAAoB,AAAC,CAAA,IAAK,gBAAgB,CAAA,eAAe,AAAC,CAAA,yBAA0B,KAAK,iBAAgB,CAAC,AAAC,CAAA,0BAA2B,KAAK,iBAAgB,CAAC,AAAC,CAAA,0BAA2B,KAAK,eAAc,CAAC,AAAC,CAAA,IAAK,cAAc,CAAA,sCAAuC,CAAA,kCAAkC,AAAC,CAAA,KAAM,WAAU,AAAC,CAAA,yBAA0B,MAAM,gBAAe,CAAC,AAAC,CAAA,YAAa,WAAU,AAAC,CAAA,yBAA0B,aAAa,gBAAe,CAAC,AAAC,CAAA,gBAAiB,mBAAkB,AAAC,CAAA,SAAU,4BAA4B,CAAA,qBAAsB,CAAA,iCAAiC,AAAC,CAAA,IAAK,eAAe,CAAA,mBAAmB,AAAC,CAAA,yBAA0B,KAAK,iBAAgB,CAAC,AAAC,CAAA,QAAS,gBAAe,AAAC,CAAA,UAAW,SAAQ,AAAC,CAAA,yBAA0B,WAAW,qBAAoB,CAAC,AAAC,CAAA,UAAW,kBAAkB,CAAA,mCAAA,CAA4B,2BAAA,CAAA,2BAA4B,CAAA,qBAAsB,CAAA,iCAAiC,AAAC,CAAA,cAAe,WAAU,AAAC,CAAA,gBAAiB,gBAAgB,CAAA,aAAc,CAAA,UAAU,AAAC,CAAA,mBAAoB,kBAAkB,CAAA,KAAM,CAAA,SAAS,AAAC,CAAA,yBAA0B,oBAAoB,WAAU,CAAC,AAAC,CAAA,2BAA4B,QAAQ,CAAA,QAAS,CAAA,uCAAA,CAA+B,mCAA/B,CAA+B,+BAAA,CAAC","file":"style.css"}
```

test.css.map output:

```
{
	"version": 3,
	"file": "style.css",
	"sources": [
		"assets/sass/style.scss",
		"assets/sass/partial/_gridtacular.scss",
		"assets/sass/partial/_rem.scss",
		"assets/sass/partial/__base.scss",
		"assets/sass/partial/__colours.scss",
		"assets/sass/partial/_links.scss",
		"assets/sass/partial/_fluid-bg.scss",
		"assets/sass/partial/_aspect-ratios.scss",
		"assets/sass/modules/_reset.scss",
		"assets/sass/modules/_grid-setup.scss",
		"assets/sass/modules/_typography.scss",
		"assets/sass/modules/_buttons.scss",
		"assets/sass/modules/_clear.scss",
		"assets/sass/modules/_forms.scss",
		"assets/sass/modules/_imgs.scss",
		"assets/sass/modules/_navigation.scss",
		"assets/sass/modules/_tables.scss",
		"assets/sass/modules/_wordpress.scss",
		"assets/sass/modules/_wireframe.scss",
		"assets/sass/section/_html-body.scss",
		"assets/sass/section/_header.scss",
		"assets/sass/section/_footer.scss",
		"assets/sass/section/_sidebar.scss",
		"assets/sass/section/_belt.scss",
		"assets/sass/section/_box.scss",
		"assets/sass/section/_billboard.scss"
	],
	"sourcesContent": [],
	"mappings": ";;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;+xBKAO,EAAE,EAAG,EAAE,CAAC,SAIX,MDMoB,QCLpB,gBAAiB,KACjB,QAAS,aACT,iBAAkB,+FAAT,EAAE,CAAC,OAGR,MDJgB,QCKhB,gBAAiB,KACjB,iBAAkB,+FGN1B,EACI,OAAQ,EACR,QAAS,EACT,SAAU,SAGN,WAAY,YAUpB,MAAO,OAAQ,OAAQ,IAAK,IAAK,MAM7B,eAAgB,QChBpB,SACI,OAAQ,QR+CJ,2BQhDR,SAGQ,MAAO,IACP,UAAW,SR4CX,2BQxCR,gBAEY,MAAO,IACP,UAAW,SRqEvB,MAEQ,YAqOa,QAxTjB,MAAM,GAAV,KAAK,QAAS,KAAK,OAEX,QAAQ,GACR,QAAQ,OAChB,KAAK,OACS,MAAM,MAgFpB,MAAQ,EAII,MAAO,KACP,SAAU,SACV,aA6NS,QApNrB,iBACQ,YAAa,GAGrB,iBAAmB,EACP,aAAc,GA2C1B,YACY,MAAO,UADnB,WAAY,YACA,MAAO,WADnB,WAAY,YACA,MAAO,KADnB,WAAY,WAAY,YACZ,MAAO,WADnB,YACY,MAAO,WADnB,WAAY,WAAY,WAAY,YACxB,MAAO,KADnB,YACY,MAAO,WADnB,WAAY,WAAY,YACZ,MAAO,WADnB,WAAY,YACA,MAAO,KADnB,WAAY,aACA,MAAO,WADnB,aACY,MAAO,WADnB,WAAY,WAAY,WAAY,WAAY,WAAY,aAChD,MAAO,MAUnB,WACgB,MAAO,KADvB,WACgB,MAAO,KADvB,WACgB,MAAO,KADvB,WACgB,MAAO,KADvB,WACgB,MAAO,MAWvB,WACgB,MAAO,WADvB,WACgB,MAAO,WADvB,WACgB,MAAO,WADvB,WACgB,MAAO,WADvB,WACgB,MAAO,WADvB,WACgB,MAAO,WADvB,WACgB,MAAO,MA6CvB,MAAQ,cACA,OAAQ,OACR,MAAO,MAzBf,YACY,YAAa,UADzB,WAAY,YACA,YAAa,WADzB,WAAY,YACA,YAAa,KADzB,WAAY,WAAY,YACZ,YAAa,WADzB,YACY,YAAa,WADzB,WAAY,WAAY,WAAY,YACxB,YAAa,KADzB,YACY,YAAa,WADzB,WAAY,WAAY,YACZ,YAAa,WADzB,WAAY,YACA,YAAa,KADzB,WAAY,aACA,YAAa,WADzB,aACY,YAAa,WAczB,aACQ,MAAO,OAmBf,IAAI,OAEI,SAAU,MACV,QAAS,MACT,OAAQ,KACR,MAAO,KACP,QAAS,SACT,UAAW,KACX,YAAa,KACb,MAAO,MACP,WAkFe,QAjFf,QAAS,SA/LT,0BAgCR,MAEQ,YAqOa,QAxTjB,MAAM,GAAV,KAAK,QAAS,KAAK,OAEX,QAAQ,GACR,QAAQ,OAChB,KAAK,OACS,MAAM,MAgFpB,MAAQ,EAII,MAAO,KACP,SAAU,SACV,aA6NS,QApNrB,oBACQ,YAAa,GAGrB,oBAAsB,EACV,aAAc,GA2C1B,eACY,MAAO,UADnB,cAAe,eACH,MAAO,WADnB,cAAe,eACH,MAAO,KADnB,cAAe,cAAe,eAClB,MAAO,WADnB,eACY,MAAO,WADnB,cAAe,cAAe,cAAe,eACjC,MAAO,KADnB,eACY,MAAO,WADnB,cAAe,cAAe,eAClB,MAAO,WADnB,cAAe,eACH,MAAO,KADnB,cAAe,gBACH,MAAO,WADnB,gBACY,MAAO,WADnB,cAAe,cAAe,cAAe,cAAe,cAAe,gBAC/D,MAAO,MAUnB,cACgB,MAAO,KADvB,cACgB,MAAO,KADvB,cACgB,MAAO,KADvB,cACgB,MAAO,KADvB,cACgB,MAAO,MAWvB,cACgB,MAAO,WADvB,cACgB,MAAO,WADvB,cACgB,MAAO,WADvB,cACgB,MAAO,WADvB,cACgB,MAAO,WADvB,cACgB,MAAO,WADvB,cACgB,MAAO,MA6CvB,MAAQ,iBACA,OAAQ,OACR,MAAO,MAzBf,eACY,YAAa,UADzB,cAAe,eACH,YAAa,WADzB,cAAe,eACH,YAAa,KADzB,cAAe,cAAe,eAClB,YAAa,WADzB,eACY,YAAa,WADzB,cAAe,cAAe,cAAe,eACjC,YAAa,KADzB,eACY,YAAa,WADzB,cAAe,cAAe,eAClB,YAAa,WADzB,cAAe,eACH,YAAa,KADzB,cAAe,gBACH,YAAa,WADzB,gBACY,YAAa,WAczB,gBACQ,MAAO,OAmBf,IAAI,OAEI,SAAU,MACV,QAAS,MACT,OAAQ,KACR,MAAO,KACP,QAAS,SACT,UAAW,KACX,YAAa,KACb,MAAO,MACP,WAkFe,QAjFf,QAAS,WA/LT,2BAgCR,MAEQ,YAqOa,QAxTjB,MAAM,GAAV,KAAK,QAAS,KAAK,OAEX,QAAQ,GACR,QAAQ,OAChB,KAAK,OACS,MAAM,MAgFpB,MAAQ,EAII,MAAO,KACP,SAAU,SACV,aA6NS,QApNrB,oBACQ,YAAa,GAGrB,oBAAsB,EACV,aAAc,GA2C1B,eACY,MAAO,UADnB,cAAe,eACH,MAAO,WADnB,cAAe,eACH,MAAO,KADnB,cAAe,cAAe,eAClB,MAAO,WADnB,eACY,MAAO,WADnB,cAAe,cAAe,cAAe,eACjC,MAAO,KADnB,eACY,MAAO,WADnB,cAAe,cAAe,eAClB,MAAO,WADnB,cAAe,eACH,MAAO,KADnB,cAAe,gBACH,MAAO,WADnB,gBACY,MAAO,WADnB,cAAe,cAAe,cAAe,cAAe,cAAe,gBAC/D,MAAO,MAUnB,cACgB,MAAO,KADvB,cACgB,MAAO,KADvB,cACgB,MAAO,KADvB,cACgB,MAAO,KADvB,cACgB,MAAO,MAWvB,cACgB,MAAO,WADvB,cACgB,MAAO,WADvB,cACgB,MAAO,WADvB,cACgB,MAAO,WADvB,cACgB,MAAO,WADvB,cACgB,MAAO,WADvB,cACgB,MAAO,MA6CvB,MAAQ,iBACA,OAAQ,OACR,MAAO,MAzBf,eACY,YAAa,UADzB,cAAe,eACH,YAAa,WADzB,cAAe,eACH,YAAa,KADzB,cAAe,cAAe,eAClB,YAAa,WADzB,eACY,YAAa,WADzB,cAAe,cAAe,cAAe,eACjC,YAAa,KADzB,eACY,YAAa,WADzB,cAAe,cAAe,eAClB,YAAa,WADzB,cAAe,eACH,YAAa,KADzB,cAAe,gBACH,YAAa,WADzB,gBACY,YAAa,WAczB,gBACQ,MAAO,OAmBf,IAAI,OAEI,SAAU,MACV,QAAS,MACT,OAAQ,KACR,MAAO,KACP,QAAS,SACT,UAAW,KACX,YAAa,KACb,MAAO,MACP,WAkFe,QAjFf,QAAS,USpPjB,EAAG,GAAI,GAAI,MAAO,SAOd,YPgDoB,IO/CpB,UPyCoB,OOxCpB,cTmTiB,QSnTrB,CAAC,YAAa,EAAE,YAAa,EAAE,YAAa,KAAK,YAAa,QAAQ,YAG9D,cAAc,GActB,GAAI,GAAI,GAAI,GAAI,GAAI,GAGhB,YP+BoB,2BO9BpB,YP2BoB,IO1BpB,cT6RiB,QS1RrB,GACI,UPUoB,QOPxB,GACI,UPOoB,QOJxB,GACI,UPIoB,QODxB,OAGI,YAAa,MAGjB,GAGI,WAAY,QAGhB,GAAI,GAIA,YT8PiB,QS3PrB,IAGI,UAAW,QAGf,UAEI,WAAY,QAGhB,YACI,WAAY,QAGhB,WACI,WAAY,OAMhB,OAAO,EAEC,UPvCgB,MQpDxB,KAAM,QACF,QAAS,aACT,MAAO,KACP,OAAQ,EACR,OAAQ,QACR,OAAQ,KACR,QAAS,KACT,sBAAuB,aAA3B,IAAI,OAAQ,OAAO,OAGX,gBAAgB,MACxB,IAAI,OAAQ,OAAO,OAGX,WAAY,MCdpB,OXMI,MAAM,GAAV,MAAM,QAAS,MAAM,OAEb,QAAQ,GACR,QAAQ,OAChB,MAAM,OACQ,MAAM,MYXpB,MAGI,QAAS,OAGb,MAAO,OAAQ,SAGX,YAAa,QACb,QAAS,MAGb,MACI,QAAS,aACT,MAAO,KACP,OAAQ,KACR,QAAS,KACT,OAAQ,gBAAZ,KAAK,OAEG,WAAY,SAIpB,SACI,MAAO,KACP,OAAQ,gBAAZ,QAAQ,OAEA,WAAY,SC3BpB,IAGI,MAAO,KACP,UAAW,KACX,OAAQ,MAGZ,EAAE,IACE,OAAQ,GGVZ,WACI,MAAO,KACP,QAAS,OAGb,YACI,MAAO,KACP,QAAS,OAGb,YACI,WAAY,KAGhB,iBACQ,MAAO,MAGf,QACI,YAAa,MAGjB,iBACI,WAAY,KAGhB,cACI,YAAa,MAGjB,aACI,OAAQ,QC7BZ,WjBII,MAAM,EiBEN,QAAS,UjBFb,UAAU,QAAS,UAAU,OAErB,QAAQ,GACR,QAAQ,OAChB,UAAU,OACI,MAAM,MiBApB,iBACQ,MAAO,KACP,MAAO,MAGf,iBACQ,MAAO,MACP,OAAQ,EACR,WAAY,MAGpB,iBACY,MAAO,KACP,QjBsSS,OiBrST,YAAa,MAGzB,iBAAiB,EACD,gBAAiB,MAGjC,QAEQ,cAAe,OACf,UAAW,QAAnB,QAAQ,EAAG,QAAQ,CAAC,SAGR,QAAS,MACT,QAAS,OACT,gBAAiB,MCvC7B,KACI,UhBuCoB,MgBnCpB,YhB6BoB,KFoBhB,0BkBtDR,KAE4C,UhBuCpB,QFahB,2BkBtDR,KAG2C,UhBuCnB,QFYhB,2BkBtDR,KAIgD,UhBuCxB,MgBnCxB,KACQ,MfLgB,QeMhB,YhBmDgB,2BgBlDhB,uBAAwB,aINhC,MACI,QAAS,ItBgDL,0BsBjDR,MAGQ,QtBqTa,SsBjTrB,aACQ,QAAS,ItByCT,0BsB1CR,aAGW,QAAS,SAIpB,iBACQ,WAAY,SAGpB,UjBjBI,kBAAmB,UACnB,gBAAiB,MACjB,oBAAqB,ekBNzB,KACI,QvB0TiB,OuBzTjB,OAAQ,cvBkDJ,0BuBpDR,KAIQ,OAAc,WAItB,SAGQ,WAAY,MAGpB,WAEQ,OAAQ,GvBoCR,0BuBtCR,WAIW,OAAc,eCnBzB,WACI,SAAU,SACV,gBAAiB,YnBGjB,kBAAmB,UACnB,gBAAiB,MACjB,oBAAqB,emBLzB,WAAW,IACF,MAAO,MlBFhB,UAAU,OAGF,YAAc,IACd,QAAS,MACT,QAAS,IkBEjB,oBACQ,SAAU,SACV,IAAK,EACL,MAAO,KxB0CP,0BwB7CR,oBAKW,MAAO,MAIlB,4BAEY,IAAK,IACL,KAAM,IACN,UAAW",
	"names": []
}
```

There is a big difference between the two. I noticed that if I do add the custom filename that it isn't reflected in the stylesheet which still has `/*# sourceMappingURL=style.css.map */`at the bottom rather than pointing to the newly named/created file.

I have also observed that if I point the CSS file to the newly named `test.css.map` that it works perfectly in Chrome. This all maybe because I am jumping the gun using features not fully documented in v1.2.3.

**Comment by am11** [comment]
This issue is fixed by v2.0.0.

**Comment by fourseven** [comment]
You released v2.0.0? SWEET! :tada: :balloon:

**Comment by am11** [comment]
Yay! Checkout the release notes: https://github.com/sass/node-sass/releases/tag/v2.0.0. :)
